misc/msvs/nmake.substitution.props

```diff
@@ -16,4 +16,12 @@
   <Target Name="Clean">
     <Exec Command="$(NMakeCleanCommandLine)"/>
   </Target>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalOptions>%(AdditionalOptions)$(AdditionalOptions)</AdditionalOptions>
```

Comment:
This won't affect consoles so it's fine by me, but why are you doing `%(AdditionalOptions)` before `$(AdditionalOptions)` here? Is there some item defined in an `ItemGroup` that has `AdditionalOptions` metadata? That's the only thing I can imagine would need this.